src/main/scala/github4s/Decoders.scala

```diff
@@ -19,4 +19,150 @@ object Decoders {
+      has_downloads ← c.downField("has_downloads").as[Boolean]
+      has_wiki ← c.downField("has_wiki").as[Boolean]
+      has_pages ← c.downField("has_pages").as[Boolean]
+      url ← c.downField("url").as[String]
+      html_url ← c.downField("html_url").as[String]
+      git_url ← c.downField("git_url").as[String]
+      ssh_url ← c.downField("ssh_url").as[String]
+      clone_url ← c.downField("clone_url").as[String]
+      svn_url ← c.downField("svn_url").as[String]
+      forks_url ← c.downField("forks_url").as[Option[String]]
+      keys_url ← c.downField("keys_url").as[Option[String]]
+      collaborators_url ← c.downField("collaborators_url").as[Option[String]]
+      teams_url ← c.downField("teams_url").as[Option[String]]
+      hooks_url ← c.downField("hooks_url").as[Option[String]]
+      issue_events_url ← c.downField("issue_events_url").as[Option[String]]
+      events_url ← c.downField("events_url").as[Option[String]]
+      assignees_url ← c.downField("assignees_url").as[Option[String]]
+      branches_url ← c.downField("branches_url").as[Option[String]]
+      tags_url ← c.downField("tags_url").as[Option[String]]
+      blobs_url ← c.downField("blobs_url").as[Option[String]]
+      git_tags_url ← c.downField("git_tags_url").as[Option[String]]
+      git_refs_url ← c.downField("git_refs_url").as[Option[String]]
+      trees_url ← c.downField("trees_url").as[Option[String]]
+      statuses_url ← c.downField("statuses_url").as[Option[String]]
+      languages_url ← c.downField("languages_url").as[Option[String]]
+      stargazers_url ← c.downField("stargazers_url").as[Option[String]]
+      contributors_url ← c.downField("contributors_url").as[Option[String]]
+      subscribers_url ← c.downField("subscribers_url").as[Option[String]]
+      subscription_url ← c.downField("subscription_url").as[Option[String]]
+      commits_url ← c.downField("commits_url").as[Option[String]]
+      git_commits_url ← c.downField("git_commits_url").as[Option[String]]
+      comments_url ← c.downField("comments_url").as[Option[String]]
+      issue_comment_url ← c.downField("issue_comment_url").as[Option[String]]
+      contents_url ← c.downField("contents_url").as[Option[String]]
+      compare_url ← c.downField("compare_url").as[Option[String]]
+      merges_url ← c.downField("merges_url").as[Option[String]]
+      archive_url ← c.downField("archive_url").as[Option[String]]
+      downloads_url ← c.downField("downloads_url").as[Option[String]]
+      issues_url ← c.downField("issues_url").as[Option[String]]
+      pulls_url ← c.downField("pulls_url").as[Option[String]]
+      milestones_url ← c.downField("milestones_url").as[Option[String]]
+      notifications_url ← c.downField("notifications_url").as[Option[String]]
+      labels_url ← c.downField("labels_url").as[Option[String]]
+      releases_url ← c.downField("releases_url").as[Option[String]]
+      deployments_url ← c.downField("deployments_url").as[Option[String]]
```

Comment:
"Historically" I have been naming parameters in the same way that Github does, in order to facilitate its use by people familiarised with the official API. I would like to keep consequently respecting this criteria along the entire project.